autoload/matchup/matchparen.vim

```diff
@@ -670,8 +670,11 @@ function! s:do_offscreen_popup_nvim(offscreen) " {{{1
           \ 'height': &previewheight,
           \ 'focusable': v:false,
           \}
-    if get(g:matchup_matchparen_offscreen, 'border', 0)
-      let l:win_cfg.border = ['', '═' ,'╗', '║', '╝', '═', '', '']
+    let l:border = get(g:matchup_matchparen_offscreen, 'border', 0)
+    if l:border != v:null
```

Comment:
I think this usually be true (unless the user passes v:null), since the default value above in the get is 0.

Reply:
Again, sorry for my lack of familiarity with Vimscript, but what I found is that checking `if l:border` failed, since strings don't seem to be truthy (which lines up with the information [here](https://learnvimscriptthehardway.stevelosh.com/chapters/21.html)). I switched to using `!empty(l:border)`, but let me know if there's a more idiomatic way to handle this case.